core/plugins/org.fusesource.ide.camel.model.service.v2_18_1/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  
+  <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <groupId>org.fusesource.ide.core</groupId>
+    <artifactId>plugins</artifactId>
+    <version>9.1.0-SNAPSHOT</version>
+    <relativePath>..</relativePath>
+  </parent>
+
+  <artifactId>org.fusesource.ide.camel.model.service.v2_18_1</artifactId>
+  <packaging>eclipse-plugin</packaging>
+  <name>JBoss Fuse Tooling :: Core :: Plugins :: Camel 2.18.1 Service Implementation</name>
+  
+  <properties>
+<!-- 
+  		<bundle.camel.version>2.18.1.fuse-000010</bundle.camel.version>
```

Comment:
I think we can remove this comment, we will create another plugin for the productized version anyway